tools/clang/include/clang/SPIRV/SpirvInstruction.h

```diff
@@ -1988,9 +2094,13 @@ class SpirvDebugDeclare : public SpirvDebugInstruction {
 
   bool invokeVisitor(Visitor *v) override;
 
+  SpirvDebugLocalVariable *getDebugLocalVariable() const { return debugVar; }
+  SpirvInstruction *getVariable() const { return var; }
+  SpirvDebugExpression *getDebugExpression() const { return expression; }
+
 private:
   SpirvDebugLocalVariable *debugVar;
-  SpirvVariable *var;
+  SpirvInstruction *var;
```

Comment:
Because we want to use it for `OpFunctionParameter` (i.e., `SpirvFunctionParameter`), it must not be `SpirvVariable`.
When initially ehsan wrote this code, he followed the spec which says this operand must be `OpVariable`.

However, I believe this must allow us to point function parameter as well in order to let a debugger inspect the value of a function parameter as it does for a local variable. This is one of spec issue I mentioned [here](https://gitlab.khronos.org/spirv/SPIR-V/issues/551).